.evergreen/csfle/activate_venv.sh

```diff
@@ -0,0 +1,25 @@
+set +x
```

Comment:
I don't think we want to change the settings in the parent shell do we?

Reply:
Removed.